R/cat_add_data.R

```diff
@@ -7,15 +7,113 @@
 #' sources.
 #'
 #' @param x an object of class [elic_cat].
-#' @param mechanism
+#' @param data_source either a [`data.frame`][base::data.frame] or
+#' [`tibble`][tibble::tibble], a string with the path to a _csv_ or _xlsx_ file,
+#' or anything accepted by the [read_sheet()][googlesheets4::read_sheet]
+#' function.
+#' @param mechanism integer indicating what mechanism the data belongs to.
 #' @inheritParams elic_cont_add_data
 #'
+#' @section Data Format:
+#'
+#' For each mechanism, data are expected to have five columns, built as follows:
+#' * The first column of the data should hold the names of the experts. The name
+#' of each expert should be repeated as many times as there are impact levels
+#' and sites considered in the elicitation
+#' (i.e. each expert should appear \eqn{number\ of\ levels
+#' \cdot number\ of\ sites} times).
+#' * The second column should be the names of the levels considered in the
+#' eliciation. Each level should be repeated as many times as there are sites
+#' considered.
```

Comment:
Lines 25-27: Here is the whole block of levels that is repeated. What about changing with:
> Each block of levels should be repeated...

or

> Each sequence of levels should be repeated...